src/reporting/utils.js

```diff
@@ -70,14 +69,18 @@ export const getSellerReport = async (conf, results) => {
  * @author Newton <cnewton@magnite.com>
  * @param {object} conf - an auction configuration object
  * @param {object} results - the results of the auction
+ * @param {object} report - the report object from the sellers report
  * @return {void} has a side effect of generating a report for the buyer
  */
 export const getBuyerReport = async (conf, results, report) => {
-	const { report_win } = await import(conf.bidding_logic_url);
+	if (!report) {
+		return null;
+	}
```

Comment:
buyer should still get a chance to send a report even if seller messed up their report

Reply:
k, fixed